package.json

```diff
@@ -37,6 +37,10 @@
     "test:playwright:recreate-tapes": "rimraf test/tapes && pnpm test:playwright:update-tapes",
     "test:playwright:update-tapes": "UPDATE_TAPES=true pnpm test:playwright",
     "test:playwright:gen": "playwright codegen localhost:8443/",
+    "test:storybook": "TEST_COMMAND=test:storybook:local ./bin/playwright.sh",
+    "test:storybook:local": "playwright test -c test/storybook",
+    "test:storybook:debug": "PWDEBUG=1 pnpm test:storybook:local",
+    "test:storybook:gen": "playwright codegen localhost:54000/",
```

Comment:
Do you have a suggestion of something else to call it?

I wanted to avoid doing something like `test:nuxt-playwright` and `test:storybook-playwright`. Mostly just for how verbose it is.

Reply:
Very valid points. What are your thoughts on
- continuing to use just `playwright` for the Nuxt tests because
  - the Openverse frontend is the main output of the repo, so any commands refer to it
  - all other commands pertaining to Nuxt don't have a `nuxt-` prefix either.
- using `sb:playwright` for the Storybook tests because
  - 'sb' is a commonly accepted abbreviation for Storybook (their own docs use it in multiple places)
  - we can also nest this under `sb` (along with `sb:build` and `sb:smoke`) making this a whole separate section of `package.json` scripts

Reply:
I think the `test` prefix is useful for discovery within package.json, maybe `test:sb-playwright`?